### DISCUSSION TITLE
DoH not working

### DISCUSSION THREAD

**Discussion by YgzNsl** [AdguardTeam/AdGuardHome#7504] [General]
I've just installed AdGuardHome in my VPS using the provided installation script.
I've valid SSL certificates from Let's Encrypt, and I enabled encryption via admin interface.
I'm not using any reverse proxies.
According to the docs, these should be enough for dns-over-https to work. However, it gives me 400 bad request.

Is there any additional config I must do?

PS: Live website can be checked here: https://dns.yagizunsal.tr/dns-query

**Comment by smurfsurfing** [comment]
once you have the two pem files.   Proceed to configure AGH --> settings / encryption settings, then set the cert path or paste cert in the two boxes.  Next just configure your devices to use DoH.  For ex. MS Edge
Settings / Privac.... / Use Secure DNS.... / Choose service provider and put in https://dns.yagizunsal.tr/dns-query

That's it

**Comment by JanFPV** [comment]
Same problem here
